Anomaly/Defs/DamageDefs/DamageDefs_Anomaly.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Defs>
+
+	<DamageDef ParentName="Nerve">
+		<defName>NerveBioferrite</defName>
+		<additionalHediffs>
+			<li>
+				<hediff>BioferriteToxicity</hediff>
+				<severityPerDamageDealt>0.05</severityPerDamageDealt>
+			</li>
+		</additionalHediffs>
+	</DamageDef>
+
+	<DamageDef ParentName="Flame">
+		<defName>BioferriteFlame</defName>
```

Comment:
Assuming we're not too concerned about it being too strong indoors/under mountains due to reduced smoke output, it might be worth adding
<explosionHeatEnergyPerCell>10</explosionHeatEnergyPerCell> (any value less than 15)
To reduce the ambient heat generation a little bit, but not as important as ramping up the regular damage, imo.

Reply:
Personally, I don't see a reason the bioferrite fuel should be "cooler" than the regular jellied chemfuel in this regard. Additionally, the risk from the heat isn't particularly high--in my testing I found you had to firing 5 - 6 full tanks from the Incinerator in a comparatively confined space even to get to 150+ F.

I think that is a reasonable margin to expect the player to take _some_ kind of precautions against burning themselves out if they're going to be using the flamer indoors enough to pose a risk. Comparatively, even 1 or 2 flaming puddles from the other types of fuel will quickly exceed those temperatures and potentially super-heat the air.